### PR TITLE
Release calypso-build@4.2.0

### PR DESCRIPTION
**Pull request by blowery**
#### Changes proposed in this Pull Request

* Bump the version on calypso-build in prep for releasing to NPM

#### Testing instructions

* Follow on from https://github.com/Automattic/wp-calypso/pull/35898, needed for https://github.com/Automattic/wp-calypso/pull/37031

### CODE REVIEW

**Review by sirreal — APPROVED** [review]
I think minor version bump is good, CHANGELOG already appears to include some entries: https://github.com/Automattic/wp-calypso/blob/master/packages/calypso-build/CHANGELOG.md

**Comment by blowery** [comment]
@sirreal updated the changelog. thanks for the push!

**Comment by blowery** [comment]
and `@automattic/calypso-build@4.2.0` is published

File: packages/calypso-build/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
+# 4.3.0
```
Comment (sirreal) [inline packages/calypso-build/CHANGELOG.md]: 😆 Should be merged with 4.2.0 below

**Pull request merged**